code/game/machinery/recharger.dm

```diff
@@ -163,3 +168,90 @@
 			icon_state = "recharger2"
 		return
 	icon_state = "recharger0"
+
+/obj/machinery/recharger/wallrecharger
+	name = "wall recharger"
+	icon = 'icons/obj/wallcharger.dmi'

```

Comment:
Since you've moved both charger icons to a new file this needs to be changed in the parent, not here

Reply:
I created a different file for the wall rechargers, not just the rechargers

Reply:
Then move the regular ones as well.